datapreview.tex

```diff
@@ -1,19 +1,20 @@
 \section{Data Previews and Data Release 1} 
 \label{sec:datapreview}
 
-A series of three Data Previews (DP) are planned  based on commissioning data to support the community as they develop their LSST analysis software and worfklows, and to enable high-impact science as soon as possible.
+A series of three Data Previews (DP) are planned to support the community as they develop their LSST analysis software and workflows, and to enable high-impact science as soon as possible.
 \begin{itemize}
 \item Data Preview 0 (DP0): Based on simulated LSST-like data.
-\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with LSSTCam.
+\item Data Preview 1 (DP1): Based on a subset of early science-grade commissioning data taken with either ComCam or LSSTCam.
 \item Data Preview 2 (DP2): Based on a full reprocessing of all science-grade LSSTCam data taken during commissioning.
 \end{itemize}
 
-Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to full a Data Release, however all Data Preview data products will be in the same science data model format as for future Data Releases.
+As both DP1 and DP2 will be based on commissioning data, they will not be released until after at least the 30-day embargo period for commissioning data has passed.
+See \citeds{DMTN-199} for further details on the data embargo periods.
+Due to the relatively short time periods available for commissioning observations (\S~\ref{ssec:scenarios}), these Data Previews will necessarily be limited in their area and temporal coverage relative to a full Data Release, however all Data Preview data products will be in the same science data model format as for future LSST Data Releases.
```

Comment:
I think the statement "same science data model format as for future LSST Data Releases" is way too strong.  At most we should say "as for Data Release 1".  

There is no reason to use RTN-011 to make a long-range commitment like this.  I think it's highly likely that we'll learn things in the first year that will result in DR2 having a substantially improved data model compared to DR1 and the Data Previews.

I'm even going to suggest that we say something weaker still, like "Data Preview 1 and 2 data products will conform to the science data model planned for Data Release 1, though we reserve the right to then refine the DR1 data model based on project and user experience with the Data Previews."

Reply:
Agree, we need flexibility for the data model to evolve. My goal was more to communicate that the DP1/2 data products would be representative of what users would get in future DRs and hence useful to start to work with. I'll rephrase

Reply:
Removed completely  and simply stated that each release/preview will come with a specific DPDD